src/components/organisms/EarthEditor/Header/hooks.ts

```diff
@@ -88,6 +88,7 @@ export default () => {
     setTeam({
```

Comment:
When using `SetTeam` why not passing the whole team object as you did in src/components/organisms/Dashboard/hooks.ts (for consistency)

Reply:
I was thinking since team includes assets (which can be large) and other things having the Header update those when changing workspaces might not be great. I'll check if there are any issues with wrong assets, etc

Reply:
So inside `src/components/organisms/Dashboard/hooks.ts` file, is it better not to pass the whole object?
or there are there no assets in this object?

https://github.com/reearth/reearth-web/blob/414bf525a6240a7411cbcf09c1a10863d9d14e52/src/components/organisms/Dashboard/hooks.ts#L51-L54